py-doc-to-xml/docx_to_xml/util.py

```diff
@@ -3,20 +3,28 @@
 
 
 def is_semantic_domain_number(s: str) -> bool:
-    """Checks if string consists of 1-5 single digits separated by decimal points."""
-    return re.match(r"^\d(\.\d){0,4}$", s) is not None
-#    return re.match(r"^[0-9.]*$", s) is not None
+    """
+    Checks if string consists of 1-5 single digits separated by decimal points.
+    There may also be a final period.
```

Comment:
For multi-line Python docstrings, the first line should be a summary (which this one is) and there should be a blank line between the summary line and subsequent text,